src/main/java/org/junit/experimental/max/MaxCore.java

```diff
@@ -114,7 +114,7 @@ public Runner getRunner() {
                     return new Suite((Class<?>) null, runners) {
                     };
                 } catch (InitializationError e) {
-                    return new ErrorReportingRunner(null, e);
+                    return new ErrorReportingRunner((Class<?>) null, e);
```

Comment:
This is a possible regression for external uses of `ErrorReportingRunner`, too? Better to use a "named constructor", a.k.a. factory method?